src/ucp/wireup/wireup.c

```diff
@@ -1144,7 +1135,7 @@ static void ucp_wireup_msg_dump(ucp_worker_h worker, uct_am_trace_type_t type,
         p += strlen(p);
 
         for (lane = 0; lane < UCP_MAX_LANES; ++lane) {
-            if (msg->tli[lane] == addr_index) {
+            if (msg->tli[lane] == (ae - unpacked_address.address_list)) {
```

Comment:
since we have `ucp_unpacked_address_for_each`, let's add `ucp_unpacked_address_index(ae)`

Reply:
done